src/main/java/org/kohsuke/github/GHPullRequestReview.java

```diff
@@ -46,6 +46,7 @@
     private String commit_id;
     private GHPullRequestReviewState state;
     private String submitted_at;
+    private String html_url;
```

Comment:
The ObjectMapper configuration explicitly declares `snake_case` strategy, but there is no single standard of field naming - I see both standard java (`htmlUrl`) and snake case (`html_url`) over the project (only fields, not getters for sure). This is confusing, IMHO it should be better to standardize it (personal preference - to java).

Reply:
@seregamorph "standardize to java naming" - agreed. It just hasn't been a priority. Especially since we don't have 
 sufficient code coverage to have high confidence in tests ensuring it is done right.  PR welcome.